src/components/organisms/OtherMember.vue

```diff
@@ -0,0 +1,99 @@
+<template>
+  <section id="member" class="member">
```

Comment:
```suggestion
  <section id="member" class="other-member">
```

ドキュメント化されていなくて申し訳ないのですが、コンポーネント名 `OrderMember` とあわせてクラス名も `other-member` にしたいです。

理由は、親コンポーネントから子コンポーネントの root の CSS に影響を与える場合があったときに、コンポーネント名と root の CSS のクラスが揃っていたほうがわかりやすいためです。

Ref: https://ja.vuejs.org/api/sfc-css-features.html#scoped-css

（そもそもなるべく子コンポーネントのスタイル変更は Props 経由で行うべきという話はありますが…